spec/api_spec.rb

```diff
@@ -2,17 +2,21 @@
 require 'orchestrate'
 
 describe HRLH::API do
-  let(:database) {double('database')}
-  let(:client) {double('client')}
-  let(:collection) {double('collection')}
+  let(:database) { double('database') }
+  let(:client) { double('client') }
+  let(:collection) { double('collection') }
+  let(:interviewer_record_in_db) { double(value: {"name" => 'Jichao', "email" => 'jichao@thoughtworks.com'}, key: key) }
+  let(:interviewer) { {name: 'Jichao', email: 'jichao@thoughtworks.com'} }
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.